javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,93 @@
+    //console.log("hello");
+   /* let yearOfBirth=document.getElementById("dateOfBirth").value ;
+    console.log(yearOfBirth);*/
+
+    /****************************************Future Age Calculator****************************************************** */
+                   
+                        /* ************************Human****************************/
+
+    function calculateAge(){
+
+            let yearOfBirth=document.getElementById("dateOfBirth").value ;
+            let yearOfFuture=document.getElementById("futureAge").value ;
+            let age =yearOfFuture- yearOfBirth;
+            let text ="You will be "+age+" years old in "+ yearOfFuture+ ".";
+            document.getElementById("ageCount").innerHTML=text;
+        
+
+    }
+
+                        /* ************************Dog****************************/
+
+    function calculateDogAge(){
+        let dogYearOfBirth = document.getElementById("dogDateOfBirth").value ;
+        let dogYearFuture = document.getElementById("dogFutureAge").value ;
+        let adogYear = dogYearFuture -dogYearOfBirth ;
+        let shouldShowResultInDogYears=document.getElementById("isDogYears").checked;
+        //console.log(adogYear);
+        
+        if(shouldShowResultInDogYears){
+            //console.log(shouldShowResultInDogYears);
+            adogYear *=5;
+            let text1="Your dog will be "+adogYear+" dog years old in "+dogYearFuture;
+            document.getElementById("dogAgeCount").innerHTML=text1;
+
+           }else{
+            let text1="Your dog will be "+adogYear+" human years old in "+dogYearFuture;
+            document.getElementById("dogAgeCount").innerHTML=text1;
+
+           }
+        }
+/* *********************************************House Price Estimator************************************************************ */
+
+                                 /* ************************Peter****************************/
+
+        let widthPeter =8;
+        let deepPeter = 10;
+        let heightPeter =10;
+        let gardenSizeInM2=100;
+        let hunderedM2Price =2500000;
+        let volumeInMeters =widthPeter*deepPeter*heightPeter;
+        let housePricePeter = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+        //console.log(housePricePeter);
+        if( hunderedM2Price>=housePricePeter){
+            let text3 ="Actural price for 100m2 house is "+hunderedM2Price+" <br>Peter is paying "+housePricePeter+ " which much less .";
+            document.getElementById("peterPrice").innerHTML=text3;
+        }else{
```

Comment:
Won't the logged message be a bit confusing if Peter paid just the right amount for the house?

Reply:
here i should use one more check for right ammont?

Reply:
Yes, exactly.